packages/terra-core-docs/src/terra-dev-site/doc/action-header/ChangeLog.3.doc.mdx

```diff
@@ -0,0 +1,6 @@
+import { Badge } from 'terra-action-header/package.json?dev-site-package';
```

Comment:
Not sure why this file shows up as a new file being added. I would think it should be shown as a file that is just moved to a different location similar to https://github.com/cerner/terra-core/pull/3431/files#diff-3ed4d041ad6d8833ce057565742867826eec7fef430f2f612042149c25337704?

Reply:
yeah it was moved, but i changed it just enough for git to not want to mark it as moved.